web/webkit/src/main/scala/net/liftweb/http/SecurityRules.scala

```diff
@@ -111,7 +111,7 @@ object ContentSourceRestriction {
   }
   /**
    * Indicates content from the given host path is allowed. See the
-   * `Content-Security-Policy` spec's [[http://www.w3.org/TR/CSP/#matching
+   * `Content-Security-Policy` spec's [[https://www.w3.org/TR/CSP/#source-list-path-patching
```

Comment:
There is nearly no measure for how entertained I am by the fact that this typo is a correct link :p